test/suite/test_cc06.py

```diff
@@ -45,8 +45,8 @@ class test_cc06(test_cc_base):
     ]
     scenarios = make_scenarios(format_values)
 
-    def get_stat(self, stat):
-        stat_cursor = self.session.open_cursor('statistics:')
+    def get_stat(self, stat, uri = ""):
```

Comment:
Can we put this definition in the `test_cc_base` class and remove the def in cc06 and cc07?

Reply:
We can remove this definition now it is defined in the base class